src/senaite/patient/content/patient.py

```diff
@@ -430,8 +436,7 @@ def setMRN(self, value):
             return
 
         # Check if a patient with this same MRN already exists
-        if patient_api.get_patient_by_mrn(value, full_object=False,
-                                          include_inactive=True):
+        if not patient_api.is_mrn_unique(value):
             raise ValueError("Value is not unique: {}".format(value))
```

Comment:
Maybe use the same message pattern here: "Patient Medical Record Number in not unique"